packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/extensions/error_model_translations.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/widgets.dart';
+
+import '../../app_extensions.dart';
+import '../models/error/error_model.dart';
+
+extension ErrorModelX on ErrorModel {
+  /// Translate the business error to a user friendly message
+  /// based on the error type.
+  String translate(BuildContext context) {
```

Comment:
This is one of the pain-points in each project, since over time this method becomes massive. Do you have any ideas how this issue could be solved ?